oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -191,7 +191,7 @@ def _populate_narrator_image(self):
             item=self._item, file_type__file_code="image_submaster"
         ).order_by("sequence"):
             self.locations.append(
-                LocationOH(url=img.file_url, label="Image of Narrator")
+                LocationOH(url=img.file_url, label="Image of Interviewee")
```

Comment:
This change of label from Narrator to Interviewee is fine, but it's in a method called `_populate_narrator_image()`.  The method name should change as well, to reflect its purpose.